bin/frontend-test-runner

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+dropdb --if-exists posthog_e2e_test
+createdb posthog_e2e_test
+DEBUG=1 DATABASE_URL=postgresql://postgres:postgres@localhost/posthog_e2e_test python manage.py migrate
```

Comment:
Default for me locally is `postgresql://localhost:5432/posthog_e2e_test` and wasn't able to overwrite this.

Reply:
that's the right default. changed!